internal/overlord/servstate/handlers.go

```diff
@@ -369,8 +371,12 @@ func (s *serviceData) startInternal() error {
 	done := make(chan struct{})
 	go func() {
 		waitErr := s.cmd.Wait()
+		err := logWriter.Flush()
+		if err != nil {
+			logger.Noticef("Failed to flush service logs: %v", err)
```

Comment:
Nit: we've settled on "Cannot X" for log and error messages, so `"Cannot flush srevice logs: %v"`.

Reply:
Done.